charts/substra-backend/templates/rbac.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if .Values.celeryworker.rbac.enable }} 
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "substra.fullname" . }}
```

Comment:
If the SA is specific to the celeryworker I'd add a suffix to the SA name. It looks like it's a global SA the way it is now.
(If its the other way - if the SA can be used by all pods - then you might want to change celeryworker.rbac.enable for rbac.enable and make use of it on all other pods)

Reply:
Yes at the moment this SA would be used only by the worker, I will add a suffix.